cdisutils/storage.py

```diff
@@ -235,7 +235,7 @@ def __init__(self,
                  lazy=False,
                  host_aliases={},
                  stream_status=False):
-        """
+        r"""
```

Comment:
Is this making the docstring a raw string? If so, what's the reason for that?

Reply:
The pyupgrade tool probably saw something that looked like a regex in the docstring.